examples/scales/asinh_demo.py

```diff
@@ -0,0 +1,58 @@
+"""
+============
+Asinh Demo
+============
+"""
+
+import numpy
+import matplotlib.pyplot as plt
+
+# Prepare sample values for variations on y=x graph:
+x = numpy.linspace(-3, 6, 100)
+
+# Compare "symlog" and "asinh" behaviour on sample y=x graph:
+fig1 = plt.figure()
+ax0, ax1 = fig1.subplots(1, 2, sharex=True)
+
+ax0.plot(x, x)
+ax0.set_yscale('symlog')
+ax0.grid()
+ax0.set_title('symlog')
+
+ax1.plot(x, x)
+ax1.set_yscale('asinh')
+ax1.grid()
+ax1.set_title(r'$sinh^{-1}$')
+
+
+# Compare "asinh" graphs with different scale parameter "linear_width":
+fig2 = plt.figure()
+axs = fig2.subplots(1, 3, sharex=True)
```

Comment:
Labels are overlapping: 

```suggestion
axs = fig2.subplots(1, 3, sharex=True, constrained_layout=True)
```